ApplyToBecomeInternal/ApplyToBecomeInternal/Views/Shared/_Submenu.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@model ProjectViewModel;
+
+
+<nav class="moj-primary-navigation" aria-label="Primary navigation">
+
+	<ul class="moj-primary-navigation__list">
+
+		@{ if (Model.Section == "ProjectNotes")
+			{
+				<li class="moj-primary-navigation__item">
+					<a class="moj-primary-navigation__link" href="/task-list/@Model.Id" >Task List</a>
+				</li>
+
+				<li class="moj-primary-navigation__item">
+					<a class="moj-primary-navigation__link" href="/application-form/@Model.Id">Application Form</a>
+				</li>
+
+				<li class="moj-primary-navigation__item">
+					<a class="moj-primary-navigation__link" aria-current="page" href="/project-notes/@Model.Id" >Project Notes</a>
+				</li>
+			}
+			else if (Model.Section == "ApplicationForm")
```

Comment:
You don't need to duplicate the HTML for each scenario. You can output the attribute based on the model section using a ternary operator. Take a look at line 55 on `_Layout.cshtml`